packages/expo-camera/src/ExponentCamera.web.tsx

```diff
@@ -1,10 +1,20 @@
 import React, { CSSProperties } from 'react';
-import { findNodeHandle, StyleSheet, View } from 'react-native';
-
 import { CapturedPicture, NativePropsType, PictureOptions } from './Camera.types';
 import CameraModule from './CameraModule/CameraModule';
 import CameraManager from './ExponentCameraManager.web';
 
+let findNodeHandle = function() {};
+let flatten = data => data;
+let View = props => <div {...props} />;
+
+// Without react-native-web
+try {
+  const ReactNative = require('react-native');
```

Comment:
Seems like this would cause issues for bundlers that statically resolve the dependency graph when building the website if `react-native` is missing. I think we should leave this out for now and once we have a few examples of modules that can be more generalized, figure out ways to write modules that can be used both with and without RN (kind of like the discussion with Nicolas yesterday).